### PR TITLE
(docs) Update README

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Updates the README with more context about what the Form Engine is, and how to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,53 +54,71 @@ yarn add @openmrs/esm-form-engine-lib@latest
...
 ```
 
-For example, if you are working on the `openmrs-esm-form-builder` module, you would run the following command:
+Or the equivalent command on Linux:
 
 ```bash
-pwd | pbcopy # copy the path to form-engine-lib to the to the clipboard
-cd `path/to/openmrs-esm-form-builder`
-yarn link `pbpaste` # paste the path to the openmrs-form-engine-lib
+pwd -W | pbcopy # copy the path to form-engine-lib to the clipboard
```
Comment (denniskigen) [inline README.md:102]: @ibacher I seem to remember us suggesting using `xclip` at some point. Not sure if the command I've suggested actually works.

Comment (ibacher) [reply README.md:102]: `pwd -W` doesn't work for me on my Mac. I think `xclip` is Linux. Why were you removing the other lines here, though? `pbcopy` will just put things on the Mac clipboard (internally called "pastebin", hence `pbcopy` and `pbpaste`).

Comment (ibacher) [reply README.md:102]: Nvm... I see the other lines were moved.

**Review by ibacher — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -107,7 +131,13 @@ yarn start --sources packages/esm-form-engine-app --backend https://link-to-my-b
...
 
 ### Production
-To use the form engine as your default in your O3 instance, you will need to add this to your importmap.
+
+Note that in addition to this library, other form engine alternatives exist within the O3 ecosystem, including:
+
+- [The HTML Form Entry module](https://github.com/openmrs/openmrs-module-htmlformentry)
+- [The Angular Form Engine](https://github.com/openmrs/openmrs-ngx-formentry)
+
+To use this library as the default form engine in your O3 instance, you will need to add this to your importmap:
```
Comment (samuelmale) [inline README.md:140]: Is it fair to assert that RFE is currently the default FE lib for O3?

**Review by samuelmale — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ibacher) [reply README.md:102]: Windows has a `clip` command, but I can't find anything that would actually paste from the clipboard.

File: README.md (hunk 2 above)
Comment (denniskigen) [reply README.md]: Fixed in the latest commits.

**Pull request merged**